docs/8-automating-documentation-updates.md

```diff
@@ -178,24 +178,6 @@ git commit -m "Add documentation build action"
 git push origin main
 ```
 
-To test out our new command, make a quick change to `docs/index.rst`:
```

Comment:
Why delete this?

Reply:
Because the workflow push to main on L178 already triggers the docs workflow to start and the page deployment job. This section is too long already, we dont need to tell them to also make a change so they can observe the workflow being triggered again?